tests/integration/kubernetes/k8s-block-volume.bats

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	get_pod_config_dir
+
+	node="$(get_one_kata_node)"
+	pod_name="pod-block-pv"
+	volume_name="block-loop-pv"
+	volume_claim="block-loop-pvc"
+	ctr_dev_path="/dev/xda"
+	vol_capacity="500M"
+
+	# Create Loop Device
+	tmp_disk_image=$(exec_host "$node" mktemp --tmpdir disk.XXXXXX.img)
+	exec_host "$node" truncate "$tmp_disk_image" --size "$vol_capacity"
```

Comment:
`exec_host()` has been relocated to `lib.sh`. Please add `load "${BATS_TEST_DIRNAME}/lib.sh"`

Reply:
Thanks, done!